src/components/Hero/components/HeroDescription/HeroDescription.module.css

```diff
@@ -0,0 +1,26 @@
+.description {
+  font-family: var(--mantine-font-family);
+  margin: 0;
+  font-weight: 500;
+  width: 850px;
+  font-size: var(--mantine-font-size-2xl);
+  line-height: 1.75;
+  letter-spacing: 0.02em;
+  padding: 0 var(--mantine-spacing-xl);
```

Comment:
`  padding-inline: var(--mantine-spacing-xl);`, this way you do not need to explicitly write that 0